usaspending_api/database_scripts/matview_generator/matview_sql_generator.py

```diff
@@ -371,6 +372,18 @@ def create_monolith_file(sql_json):
     write_sql_file(sql_strings, os.path.join(DEST_FOLDER, sql_json["final_name"]))
 
 
+def add_chunk_strings(sql_json, chunk):
+    chunked_sql_json = copy.deepcopy(sql_json)
+
+    chunk_count = GLOBAL_ARGS.chunk_count
+
+    if chunk_count > 1:
+        chunked_sql_json["final_name"] += f"_{chunk}"
```

Comment:
Minor suggestion to simplify reading the line. Only todo if there are other changes.

```suggestion
        chunked_sql_json["final_name"] = f"{chunked_sql_json['final_name']}_{chunk}"
```